docs/en/blog/posts/2024-06-01-documenting-rust-code-with-markdown.md

```diff
@@ -0,0 +1,200 @@
+---
+author: Kristijan Lenković
+authors:
+  - klenkovic
+date: 2024-06-01
+readtime: 15
+tags:
+  - rust
+  - markdown
+  - documentation
+  - rustdoc
+  - axum
+  - web server
+description: >-
+  A small demonstration of the usage of Markdown while documenting simple Rust
+  web service powered by axum and generating HTML documentation with rustdoc tool.
+---
+
+# Documenting Rust code with Markdown
+
+![Happy Rustacean](https://www.rustacean.net/assets/rustacean-flat-happy.svg){ align=left style="width: 190px" }
+
+Recently, I decided to move on from [Node.js](https://nodejs.org/) and start learning something new related to web development. Over the past several years, [Rust](https://www.rust-lang.org/) has gained significant popularity, to the point where [backend web services](https://github.com/tokio-rs/axum/blob/main/ECOSYSTEM.md#project-showcase) and numerous [other extraordinary CLI tools and apps](https://github.com/rust-unofficial/awesome-rust) are being built with it.
+
+As I started learning and tinkering with Rust, I discovered a feature I wasn't aware of - the handy and powerful documentation tool called [rustdoc](https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html), which comes with the standard Rust distribution. What's cool about `rustdoc` is that it uses Markdown syntax, allowing developers to document their code with Markdown and generate HTML documentation. Think of it as "*MkDocs meets Doxygen.*"
+
+Since the [esteemed members](../../people/index.md) of the **😎 GASERI** group [love their Markdown](../../../hr/aktivizam/mapapijri.md), I thought it would be nice to share this lovely "discovery" with the rest of the community by demonstrating how to document a "Hello World" Rust web service.
+
+<!-- more -->
+
+## Installation
+
+First, we must ensure that Rust is installed on our machine. The recommended way is to use [rustup](https://rust-lang.github.io/rustup/), the Rust toolchain installer. As mentioned, `rustdoc` is part of the standard Rust distribution and will be installed with `rustup`, along with the [rustc](https://doc.rust-lang.org/rustc/what-is-rustc.html) compiler and [cargo](https://doc.rust-lang.org/cargo/) package manager. You can get `rustup` on [the Rust website here](https://www.rust-lang.org/tools/install) or follow [the official guide for other installation methods here](https://forge.rust-lang.org/infra/other-installation-methods.html).
+
+[Install :fontawesome-brands-rust:](https://www.rust-lang.org/tools/install){ .md-button .md-button--primary }
+
+## Project Setup
+
+Once you have installed Rust on your system, let's create a new project and install some dependencies that we will need for this to work.
+
```

Comment:
Would be nice to have a paragraph here giving a bit of context, i.e. Axum is ... Tokio is ... We'll install them right away.